src/auditwheel/policy/__init__.py

```diff
@@ -201,9 +200,6 @@ def filter_libs(
                     # 'ld64.so.1' on ppc64le
                     # 'ld-linux*' on other platforms
                     continue
-                if LIBPYTHON_RE.match(lib):
-                    # always exclude libpythonXY
-                    continue
```

Comment:
This has the effect of checking version symbols in libpython which we don't want to do.

Reply:
This is fixed with the latest commits.
While a test has been added to check that a wheel with libpython is repaired to the correct glibc limit, the test  depends on the environment for failure...
Checked manually inside a `ghcr.io/mayeut/manylinux_2_35:2025.05.19-1` container.